sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -416,9 +416,9 @@ case class DropTable(
 }
 
 /**
- * The logical plan for handling non-existing table for DROP TABLE command.
+ * The logical plan for no-op command handling non-existing table.
  */
-case class NoopDropTable(multipartIdentifier: Seq[String]) extends Command
+case class NoopCommand(multipartIdentifier: Seq[String]) extends Command
```

Comment:
We should probably add a `commandName: String` property, which can be `DROP TABLE`, `REFRESH TABLE`, etc., so that we can see the original commannd name from the EXPLAIN result.

Reply:
Added `commandName`. Now `EXPLAIN EXTENDED DROP TABLE` looks like the following:
```
|== Parsed Logical Plan ==
'DropTable true, false
+- 'UnresolvedTableOrView [testcat, ns1, ns2, tbl], true

== Analyzed Logical Plan ==
NoopCommand DROP TABLE, [testcat, ns1, ns2, tbl]

== Optimized Logical Plan ==
NoopCommand DROP TABLE, [testcat, ns1, ns2, tbl]

== Physical Plan ==
LocalTableScan <empty>
```

Btw, do we want to introduce `NoopCommandExec` for physical plan as well?